src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -122,7 +123,7 @@ private DistributedApplicationExecutionContextOptions BuildExecutionContextOptio
         var operation = _innerBuilder.Configuration["AppHost:Operation"]?.ToLowerInvariant() switch
         {
             "publish" => DistributedApplicationOperation.Publish,
-            "run" => DistributedApplicationOperation.Run,
+            "run" or "exec" => DistributedApplicationOperation.Run,
```

Comment:
I'm not sure about the value of having another operation. Doesn't the presence of the commandline argument signal that we are doing a resource exec - given we interpret `exec` as `DistributedApplicationOperation.Run` anyway - it seems redundant.